sass/_CategoryFacetSearch.scss

```diff
@@ -82,10 +81,20 @@
 .coveo-category-facet-search-path {
   display: block;
   font-size: 11px;
+  @include display(flex);
 }
 
 .coveo-category-facet-search-path-parents {
   color: $color-blueish-gray;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding-right: 5px;
+  direction: rtl;
```

Comment:
Hmm, why right-to-left here?

Reply:
Played with both and talked with Lucille a bit, ultimately it's better to see this : 
```
... the/last/part/of/the/path  TheValue
```
instead of 
```
the/beginning/of/the/path ... TheValue
```